compiler/optimizer/Inliner.cpp

```diff
@@ -1683,7 +1684,7 @@ void TR_InlinerBase::rematerializeCallArguments(TR_TransformInlinedFunction & ti
             if (rematTree == argStoreTree)
                {
                TR::Node *duplicateStore = argStore->duplicateTree();
-               if (performTransformation(comp(), "O^O GUARDED CALL REMAT: Rematerialize [%p] as [%p]\n", argStore, duplicateStore))
+               if (performTransformation(comp(), "%sRematerialize [%p] as [%p]\n", OPT_DETAILS_GUARDED_CALL_REMAT, argStore, duplicateStore))
```

Comment:
We can reword the message to indicate that a guarded call is being rematerialized.